htc-api/api/htc_api.py

```diff
@@ -32,6 +32,7 @@
 global pool
 global log
 
+FACTID_DIABETES = 1;
```

Comment:
There's no guarantee that diabetes has a fact ID of 1. You should be querying the synth_ma.synth_disease table to figure out what the diseasefp is. Each stat we track has a "stat_name". So, this query may look like:

SELECT diseasefp FROM synth_ma.synth_disease WHERE stat_name = 'diabetes';